wolfcrypt/src/evp.c

```diff
@@ -6878,6 +6878,446 @@ int wolfSSL_EVP_get_hashinfo(const WOLFSSL_EVP_MD* evp,
 }
 #endif /* !defined(NO_PWDBASED) */
 
+/* Base64 encoding APIs */
+#if defined(WOLFSSL_BASE64_ENCODE) || defined(WOLFSSL_BASE64_DECODE)
+static struct WOLFSSL_EVP_ENCODE_CTX* wolfSSL_EVP_ENCODE_CTX_new_ex(void* heap);
+/*  wolfSSL_EVP_ENCODE_CTX_new allocates WOLFSSL_EVP_ENCODE_CTX
+ *  Returns WOLFSSL_EVP_ENCODE_CTX structure on success, NULL on failure.
+ */
+struct WOLFSSL_EVP_ENCODE_CTX* wolfSSL_EVP_ENCODE_CTX_new(void)
+{
+    return wolfSSL_EVP_ENCODE_CTX_new_ex(NULL);
+}
+/* wolfSSL_EVP_ENCODE_CTX_new_ex is a helper function of
+ * wolfSSL_EVP_ENCODE_CTX_new to allocate WOLFSSL_EVP_ENCODE_CTX structure.
+ */
+static struct WOLFSSL_EVP_ENCODE_CTX* wolfSSL_EVP_ENCODE_CTX_new_ex(void* heap)
+{
+    WOLFSSL_EVP_ENCODE_CTX* ctx = NULL;
+    WOLFSSL_ENTER("wolfSSL_EVP_ENCODE_CTX_new");
+    ctx = (WOLFSSL_EVP_ENCODE_CTX*)XMALLOC( sizeof(WOLFSSL_EVP_ENCODE_CTX),heap,DYNAMIC_TYPE_OPENSSL );
```

Comment:
Please use spaces after commas, wrap lines that are longer than 80 characters.  Apply space after comma comment to other areas of this PR as well.

Reply:
Fixed them.